### PR TITLE
add recycling facts

### PR DESCRIPTION
**Pull request by AndyCErnst**
For cycling on the front page. Feel free to add more, but keep track of where they are from so we can credit them.

### CODE REVIEW

File: src/data/recycling-facts.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
+[
+  "Recycling is a $200 billion industry in the U.S.",
+  "Recycling generates 7-10 more jobs that landfills and waste to energy plants.",
```
Comment (adamosimpson21) [inline src/data/recycling-facts.json]: should this be "Recycling generates 7-10 more jobs THAN landfills?

File: src/data/recycling-facts.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  "Recycling properly creates valuable resources for U.S. manufacturing and can become a highly valuable export to countries such as China and India.",
+  "Recycling conserves finite natural resources - this is critical as ​population continues to grow exponentially.",
+  "Recycling conserves fresh water up to 95% in the mining and manufacturing process for many materials.",
```
Comment (adamosimpson21) [inline src/data/recycling-facts.json]: I think what this is trying to say is "Recycling conserves fresh water, up to 95%, during the mining and manufacturing process..."

**Review by adamosimpson21 — COMMENTED** [review]
There are a handful of typos in these facts, and it seems like a lot of commas got omitted, I'm not sure if we want to do copy-editing for these kinds of things, but the list seems solid. Are you keeping track of the sources through the contributors page, or the acknowledgements page? Either seem possible.

**Comment by AndyCErnst** [comment]
Thanks for reading through them. I pulled them from two different websites and already made a few grammatical fixes... so they obviously weren't written very carefully. Yes, we should edit them to improve clarity.

**Review by adamosimpson21 — APPROVED** [review]
Awesome! Looks like the grammatical stuff has been cleared up. Any other small changes we'll get to at a later date

**Pull request merged**